resolwe_api/resolwe.py

```diff
@@ -245,19 +277,11 @@ def create(self, data, resource='data'):
             data = json.dumps(data)
 
         if not isinstance(data, str):
```

Comment:
Does this work with Python3? Write a test. In general, we should use `six.string_types` instead of `str` to support Python3.